osf-cas/templates/configmap.yaml

```diff
@@ -319,6 +319,352 @@ httpd/conf.d/shib.conf: |-
+          <Property name="apereo.service.log.level">warn</Property>
+          <Property name="apereo.inspektr.log.level">info</Property>
+          <!-- Spring framework logging levels -->
+          <Property name="spring.boot.admin.log.level">warn</Property>
+          <Property name="spring.boot.log.level">info</Property>
+          <Property name="spring.cloud.log.level">warn</Property>
+          <Property name="spring.security.log.level">info</Property>
+          <Property name="spring.web.log.level">debug</Property>
+          <Property name="spring.webflow.log.level">debug</Property>
+          <!-- Logging levels for 6.2.x packages that are explicitly used by OSF CAS -->
+          <Property name="thymeleaf.log.level">info</Property>
+          <Property name="pac4j.log.level">info</Property>
+          <!-- Logging levels for new packages that are added by OSF CAS -->
+          <Property name="hibernate.log.level">warn</Property>
+          <Property name="scribejava.log.level">warn</Property>
+          <Property name="google.gson.log.level">warn</Property>
+          <!-- Generic logging levels for 6.2.x packages that are not explicitly used by OSF CAS -->
+          <Property name="generic.off.log.level">off</Property>
+          <Property name="generic.error.log.level">error</Property>
+          <Property name="generic.warn.log.level">warn</Property>
+          <Property name="generic.info.log.level">info</Property>
+          <Property name="generic.debug.log.level">debug</Property>
+      </Properties>
+      <Appenders>
+          <Console name="console" target="SYSTEM_OUT">
+              <PatternLayout pattern="%highlight{%d %p [%c] - &lt;%m&gt;}%n"/>
+          </Console>
+          <RollingFile name="file" fileName="${baseDir}/cas.log" append="true"
+                      filePattern="${baseDir}/cas-%d{yyyy-MM-dd-HH}-%i.log">
+              <PatternLayout pattern="%d %p [%c] - &lt;%m&gt;%n"/>
+              <Policies>
+                  <OnStartupTriggeringPolicy/>
+                  <SizeBasedTriggeringPolicy size="10 MB"/>
+                  <TimeBasedTriggeringPolicy/>
+              </Policies>
+          </RollingFile>
+          <RollingFile name="auditlogfile" fileName="${baseDir}/cas_audit.log" append="true"
+                      filePattern="${baseDir}/cas_audit-%d{yyyy-MM-dd-HH}-%i.log">
+              <PatternLayout pattern="%d %p [%c] - %m%n"/>
+              <Policies>
+                  <OnStartupTriggeringPolicy/>
+                  <SizeBasedTriggeringPolicy size="10 MB"/>
+                  <TimeBasedTriggeringPolicy/>
+              </Policies>
+          </RollingFile>
+          <CasAppender name="casAudit">
+              <AppenderRef ref="auditlogfile"/>
+          </CasAppender>
+          <CasAppender name="casFile">
+              <AppenderRef ref="file"/>
+          </CasAppender>
+          <CasAppender name="casConsole">
+              <AppenderRef ref="console"/>
+          </CasAppender>
```

Comment:
For now, only these three appenders are used. Sentry is missing which will be added later.